### PR TITLE
[RPA-1260.4]  Show admin all collections in tree.

### PR DESCRIPTION
**Pull request by star-in-the-jar**
…tions in tree & change runbotics user password.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

**Review by star-in-the-jar — COMMENTED** [review]
self review

File: runbotics-orchestrator/src/main/java/com/runbotics/service/impl/ProcessCollectionServiceImpl.java [hunk 1]
```diff
@@ -138,9 +138,18 @@ public ProcessCollectionDTO save(ProcessCollectionDTO processCollectionDTO) {
...
+        if (hasUserAccessEveryCollection) {
+            List<ProcessCollection> allCollections = processCollectionRepository.getAll();
+            log.debug("allCollections: {}", allCollections);
+
+            return processCollectionMapper.toDto(allCollections);
+        } else {
+            List<ProcessCollection> userAccessible = processCollectionRepository.findAllUserAccessible(currentUser);
+            log.debug("userAccessible: {}", userAccessible);
+            return processCollectionMapper.toDto(userAccessible);
+        }
```
Comment (marcin-depcik) [inline runbotics-orchestrator/src/main/java/com/runbotics/service/impl/ProcessCollectionServiceImpl.java:153]: you can remove `else` here

File: runbotics-orchestrator/src/main/resources/config/liquibase/changelog/20240409121600_changelog.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-4.1.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet author="jarockis" id="20240409121600-1">
+        <sql>
+            UPDATE jhi_user
+            SET password_hash = '$2y$10$w9UaGt/hmubP3Iy8lQdy0uFS6HIGCQSDVD.nJWwxa/mhzJ9Rg5CT.'
+            WHERE id = 3
+        </sql>
+    </changeSet>
+</databaseChangeLog>
```
Comment (marcin-depcik) [inline runbotics-orchestrator/src/main/resources/config/liquibase/changelog/20240409121600_changelog.xml:10]: I would add check if user with id 3 exists

**Review by marcin-depcik — CHANGES_REQUESTED** [review]

File: runbotics-orchestrator/src/main/resources/config/liquibase/changelog/20240409121600_changelog.xml (hunk 2 above)
Comment (star-in-the-jar) [reply runbotics-orchestrator/src/main/resources/config/liquibase/changelog/20240409121600_changelog.xml:10]: okazuje sie, że preconditions można pozostawić puste - źródło [Preconditions (liquibase.com)](https://docs.liquibase.com/concepts/changelogs/preconditions.html)

File: runbotics-orchestrator/src/main/java/com/runbotics/service/impl/ProcessCollectionServiceImpl.java (hunk 1 above)
Comment (star-in-the-jar) [reply runbotics-orchestrator/src/main/java/com/runbotics/service/impl/ProcessCollectionServiceImpl.java:153]: I could, ale czy mi się chce? xD

**Pull request merged**